controllers/reconcilers/logging_installation/logging_installation_reconciler.go

```diff
@@ -371,7 +373,7 @@ func (r *Reconciler) createClusterLogForwarderCr(ctx context.Context, cr *v1.Obs
 					OutputTypeSpec: v14.OutputTypeSpec{
 						Cloudwatch: &v14.Cloudwatch{
 							Region:  "eu-west-1",
-							GroupBy: "namespaceName",
+							GroupBy: "logType",
```

Comment:
envision to make these configurable

Reply:
Yeah I agree, I'll create a ticket to make these more easily configurable through the config repo.

Reply:
Thank you